src/scraper.py

```diff
@@ -0,0 +1,163 @@
+import sys
+import requests
+import re
+from bs4 import BeautifulSoup
+import io
+
+INSPECTION_DOMAIN = 'http://info.kingcounty.gov'
+INSPECTION_PATH = '/health/ehs/foodsafety/inspections/Results.aspx'
+INSPECTION_PARAMS = {
+    'Output': 'W',
+    'Business_Name': '',
+    'Business_Address': '',
+    'Longitude': '',
+    'Latitude': '',
+    'City': '',
+    'Zip_Code': '',
+    'Inspection_Type': 'All',
+    'Inspection_Start': '',
+    'Inspection_End': '',
+    'Inspection_Closed_Business': 'A',
+    'Violation_Points': '',
+    'Violation_Red_Points': '',
+    'Violation_Descr': '',
+    'Fuzzy_Search': 'N',
+    'Sort': 'H'
+}
+
+
+def get_inspection_page(**kwargs):
+    """ Makes a request to the King County Server, fetches
+    search results, takes query parameters as arguments, returns the
+    content and encoding, if problem raises an error.
+    """
+
+    url = INSPECTION_DOMAIN + INSPECTION_PATH
+    params = INSPECTION_PARAMS.copy()
+    for key, val in kwargs.items():
+        if key in INSPECTION_PARAMS:
+            params[key] = val
+    response = requests.get(url, params=params)
+    response.raise_for_status()
+    return response.content, response.encoding
+
+
+def load_inspection_page(**kwargs):
+    """ The function: reads the inpection_page.html file; returns
+    the content and encoding.
+    """
+
+    files = io.open(paths, 'response_body')
```

Comment:
That's not quite how `io.open` works -- the second argument needs to be the mode, e.g. `r` (read), `w` (write), `rb` (read bytes) etc.